EventListener/RouteAnnotationEventListener.php

```diff
@@ -96,6 +113,14 @@ private function addUrlsFromRoutes(SitemapPopulateEvent $event)
                 continue;
             }
 
+            if ($this->defaultOptions['default_locale']) {
+                if (strpos($name, $this->defaultOptions['default_locale']) === false) {
+                    continue;
+                }
+
+                $name = preg_replace('/[a-z]+__RG__/', '', $name);
```

Comment:
I'm not sure to understand this part, what's this for ?

Reply:
I'm use http://jmsyst.com/bundles/JMSTranslationBundle to map different url per language. New url have that structure

 it__RG__landing_home       /home                                                       
  en__RG__landing_home    /en/home                                                    
  it__RG__landing_about      /chi-siamo                                                  
  en__RG__landing_about    /en/about-us                                                
  it__RG__landing_agency    /cosa-facciamo

So to obtain "canonical_url" trim first part of route name

Reply:
So, the Pull Request you wrote is about supporting translatable URLs **with** `JMSTranslationBundle`.

But what about Symfony's I18n routes ?
- https://symfony.com/doc/current/routing.html#localized-routes-i18n
- https://symfony.com/blog/new-in-symfony-4-1-internationalized-routing ?

Reply:
Yep, i'll try to support also symfony i18n, but it isn't supported on symfony 3.4 https://symfony.com/doc/3.4/routing.html#translating-routes

Reply:
I handled both cases in my last commit